src/rating/rating.ts

```diff
@@ -17,12 +20,37 @@ export class NgbRating implements OnInit {
   private _oldRate: number;
   range: number[] = [];
 
+  /**
+   * Maximal rating that can be given using this widget.
+   */
   @Input() max: number = 10;
+
+  /**
+   * Current rating.
+   */
   @Input() rate: number;
-  @Input() readonly: boolean;
 
+  /**
+   * A flag indicating if rating could be read only (true) or updated (false).
```

Comment:
Or how about `A flag indicating if rating can be updated`. I think this is actually a little better at communicating the intent